test/wsdl-parse-test.js

```diff
@@ -14,4 +15,13 @@ describe(__filename, function () {
       done();
     });
   });
+
+  it('should only parse a WSDL once', function(done) {
```

Comment:
> I will add a test with recursive WSDLs using open_wsdl

Thank you!  I believe that would be the best approach.  If you do that, then I believe you can avoid exporting the new function used internally by `WSDL`.